src/components/Progressbar/Progressbar.tsx

```diff
@@ -0,0 +1,75 @@
+import { Progress } from '@chakra-ui/react'
+import { Text } from '@chakra-ui/react'
+import style from './Progressbar.module.css'
+
+
+interface Props {
+    /**
+     * Цвет прогресс бара
+    */
+    colorScheme?: string,
+    /**
+     * Градиент прогресс бара
+    */
+    hasStripe?: boolean,
+    /**
+     * Анимированный прогресс бара
+    */
+    isAnimated?: boolean,
+    /**
+     * Максимальное значение прогресс бара
+    */
+    max?: number,
+    /**
+     * Минимальное значение прогресс бара
+    */
+    min?: number,
+    /**
+     * Размер прогресс бара
+    */
+    size?: 'sm' | 'md' | 'lg';
```

Comment:
У прогресса в chakra UI есть еще размер `xs`
```suggestion
    size?: 'xs' | 'sm' | 'md' | 'lg';
```